app/src/ui/history/commit-summary.tsx

```diff
@@ -547,9 +548,14 @@ export class CommitSummary extends React.Component<
         case AppFileStatusKind.Deleted:
           filesRemoved += 1
           break
+        case AppFileStatusKind.Renamed:
+          filesRenamed += 1
       }
     }
 
+    const hasFileDescription =
+      filesAdded + filesModified + filesRemoved + filesRemoved > 0
```

Comment:
I think this should be…
```suggestion
      filesAdded + filesModified + filesRemoved + filesRenamed > 0
```